code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1221,6 +1221,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		. += speedmod
 		. += H.physiology.speed_mod
 
+	if (H.m_intent == MOVEMENT_INTENT_WALK && has_trait("quickie_feet"))
```

Comment:
`if (H.m_intent == MOVEMENT_INTENT_WALK && H.has_trait(TRAIT_SPEEDY_STEP))`

Also check if this isn't overridden by the subtypes.